src/main/java/Logos/subCategory/SubCategory.java

```diff
@@ -66,15 +66,32 @@ public String getDescription() {
     }
 
     public boolean isActive() {
-        return CategoryStatus.ACTIVE.equals(this.status);
+        return SubCategoryStatus.ACTIVE.equals(this.status);
     }
 
     public boolean isEmptyDescription() {
-        return this.getDescription().isEmpty();
+        return this.getDescription()!=null? this.getDescription().isEmpty():false;
```

Comment:
Que tal `this.getDescription() != null && this.getDescription().isEmpty();`? [Apenas sugestão] para uma melhoria de leitura